<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QWidget" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户信息</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QSpinBox" name="id">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QComboBox" name="status">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>正常</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>冻结</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="account">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="yes">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户角色</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QComboBox" name="role">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>68</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户修改</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户状态</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yes_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_2">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="id_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>68</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户添加</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="role_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户角色</string>
     </property>
    </widget>
    <widget class="QComboBox" name="status_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>正常</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>冻结</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="account_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="yes_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_3">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>68</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>角色修改</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="id_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>角色名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>角色名称</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_4">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="yes_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="id_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>68</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>角色添加</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
